tests/src/tests/api/storage.rs

```diff
@@ -24,13 +42,22 @@ pub struct TestApiStorageCallsNode {
 }
 
 #[test]
-fn test_get_shared_commitments() {
+fn test_get_blob_data() {
```

Comment:
Would it be possible to move these tests to `da` module (maybe split `da/{api.rs,dispersal.rs}`, and instead of `TestApiStorageCallsNode` use real node?